tree/ntuple/v7/inc/ROOT/RNTupleProcessor.hxx

```diff
@@ -300,6 +363,29 @@ public:
    static std::unique_ptr<RNTupleProcessor> CreateJoin(const std::vector<RNTupleOpenSpec> &ntuples,
                                                        const std::vector<std::string> &joinFields,
                                                        std::vector<std::unique_ptr<RNTupleModel>> models = {});
+
+   /////////////////////////////////////////////////////////////////////////////
+   /// \brief Create a new RNTuple processor for horizontally combined RNTuples.
```

Comment:
Similar comment here w.r.t. aligning the docs with other RNTupleProcessor derived classes factories.